src/tests/moods.js

```diff
@@ -0,0 +1,37 @@
+const add = require("../actions/moods/add-moods"),
+	remove = require("../actions/moods/remove-moods"),
+	specificMoods = require("../actions/moods/specific-moods"),
+	updateMoods = require("../actions/moods/update-moods"),
+	mood = require("../models").mood;
+
+describe("Moods", () => {
```

Comment:
`describe("Mood routes"`, it should describe more specifically what you're going to test. This in turn will make the test descriptions read nicer.